tests.py

```diff
@@ -311,6 +311,28 @@ def test_import_from_qtwidgets():
     assert QPushButton.__name__ == "QPushButton", QPushButton
 
 
+def test_pyside2uic():
```

Comment:
Ah, but I think that's testing the Docker image within which we run tests for Qt.py. The tests don't belong amongst tests for Qt.py. We're basically producing tests for this particular distribution of pyside2-tools that we got from some random PPA for this particular OS. It doesn't prove anything. We should make sure what we install to run our tests are correct, and do that external from these tests.